Doc/library/graphlib.rst

```diff
@@ -183,13 +186,12 @@
           >>> print([*ts2.static_order()])
           [0, 2, 1, 3]
 
-      This is due to the fact that "0" and "2" are in the same level in the
+      This is due to the fact that "0" and "2" are on the same level in the
       graph (they would have been returned in the same call to
       :meth:`~TopologicalSorter.get_ready`) and the order between them is
       determined by the order of insertion.
 
-
-      If any cycle is detected, :exc:`CycleError` will be raised.
+      Raises :exc:`CycleError` if any cycles are detected.
```

Comment:
Why change this?

Reply:
For consistency with the rest of pythons docs.
The error messages are usually written in the format:
`Raises MyError if my-problem is encountered.`

Also, I changed it to use the active voice instead of the passive voice.

Reply:
I don’t think the original phrasing was incorrect or misleading or problematic in any way.
The Python codebase is old and made by many contributors.  Not everything is fully consistent, and that is not a goal.

Reply:
Yea its correct, but its still poorly written.

The current phrasing puts the most important word "raises" at the end of the sentence.
A reader who is skimming through the page is going to have to read all the way to the end of the sentence to see what the verb is.

The new phrasing starts with the word "Raises" so that before you've even read the sentence, you already know what topic its going to describe.
The new phrase is shorter and easier to comprehend.